test/commands/lightning/dev/app.test.ts

```diff
@@ -226,24 +237,6 @@ describe('lightning dev app', () => {
       await verifyMobileThrowsWhenFailedToGenerateCert(Platform.android);
     });
 
-    /* it('waits for user to manually install the certificate', async () => {
```

Comment:
These were left over from another PR. These should have been removed b/c they no longer apply.